apps/dashboard/app/models/launcher.rb

```diff
@@ -9,7 +9,7 @@ class ClusterNotFound < StandardError; end
   attr_reader :title, :id, :created_at, :project_dir, :smart_attributes
 
   class << self
-    def scripts_dir(project_dir)
+    def launchers_dir(project_dir)
       Pathname.new("#{project_dir}/.ondemand/scripts")
```

Comment:
While internal names need to update for sure - this actual Pathname here needs to update. I.e., the directory we create here should be called `launchers` too.